content/en/blog/_posts/xxxx-xx-xx-Fine-grained-SupplementalGroups-control/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: blog
+title: 'Kubernetes 1.31: Fine-grained SupplementalGroups control'
+date: xxxx-xx-xx
+slug: fine-grained-supplementalgroups-control
+author: >
+  Shingo Omura (Woven By Toyota)
+
+---
+
+This blog discusses a new feature in Kubernetes 1.31 to improve the handling of supplementary groups in containers in Pods.
```

Comment:
I would go for "improve the handling of supplementary groups in containers deployed **within** a pod"

To avoid the "in...in" repetition. 

Just a tiny clarity change in name of the reading fluency!

Reply:
fixed in https://github.com/kubernetes/website/pull/46921/commits/a8fa614e22cd9e683a201d6c85c2d99d27be6a67